func/ui.c

```diff
@@ -13,21 +13,37 @@
 
 int getIdFromString(char *, Recipe *, int);
 
+// Function to print the list of recipes if the user wants it.
+// This is done to make it easier for the user to select the correct recipe
+void listLoadedRecipes(Recipe *recipes, int recipeCount){
+    char printList = '0';
```

Comment:
```suggestion
    char printList;
```
Very minor deet: there is no reason to initialize this with a value, since it will never be read before a valid input has been made.